src/PlantList.tsx

```diff
@@ -147,44 +153,32 @@ function PlantList() {
                 <Grid item xs={4}>
                 
                 <Tooltip title={`${formatNumberWithDecimals(netWaterRate)}/Second`}>
-                    <Box border={1} borderColor="grey.300" borderRadius={2} padding={0.2}>
-                        <Typography><OpacityIcon sx={{ fontSize: 22, color: 'blue' }} /> {formatNumberWithDecimals(plant.water)}</Typography>
-                    </Box>
+                        <Water amount={plant.water} />
```

Comment:
This line in particular is the magic. This will format all water amounts the same, with the same icon. Always. Ditto every other resource.